Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m

```diff
@@ -243,6 +267,25 @@ - (void)loadForDisplay
     OWSAssert(self.viewItem.interaction);
     OWSAssert([self.viewItem.interaction isKindOfClass:[TSMessage class]]);
 
+    if (self.hasFailedSendBadge) {
```

Comment:
naming nit: this BOOL could have a little clearer name, since it doesn't **yet** have a failedSendBadge. 

e.g. `shouldHaveFailedToSendBadge` or maybe just `hasFailedToSend`

Reply:
Done.